src/ui/screens/onboarding/OnboardingStartSlideDesktop.qml

```diff
@@ -61,11 +62,11 @@ ColumnLayout {
         MZSettingsToggle {
             objectName: "startAtBootToggle"
 
-            checked: MZSettings.startAtBoot
+            checked: startAtBootCheckbox
```

Comment:
If carting around the `startAtBootCheckbox` var makes you nervous you could also give the toggle an ID and then do: `Component.onCompleted : { toggleID.checked = true }` and then `if (checked)` on L65 and `MZSettings.startAtBoot = toggleID.checked` on L113.